.github/ISSUE_TEMPLATE/3-Feature_request.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Feature request
+about: You want something added to ASGARDEX.
+labels: 'enhancement'
+---
+
+## Description
+
+<!--- Please describe the enhancement you would like to see in a next version of ASGARDEX -->
```

Comment:
<!--- Please describe the enhancement you would like to see in a next version of ASGARDEX -->
=> (not sure)
<!--- Please describe the enhancement you would like to see at the next version of ASGARDEX -->